src/serviceClients/catalogClient.ts

```diff
@@ -31,14 +31,14 @@ export class CatalogClient extends HttpClient {
   }
 
   @withSpanAsyncV4
-  public async findByInternalId(internalId: string): Promise<FindRecordResponse> {
+  public async findByCatalogId(catalogId: string): Promise<FindRecordResponse> {
```

Comment:
In that case, i think we can use `findById(catalogId: string)` is kinda clear enough according to the fact its under 'catalogClient'

Reply:
done